_posts/2020-04-03-css-grid.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: Just Enough CSS Grid to be Dangerous
+date: 2020-04-03
+tags: css grid
+author: Tim Veletta
+image: img/blog/css-grid/named-areas.png
+---
+
```

Comment:
You probably want to restructure your introduction slightly. I'd probably start with explaining that CSS is used style and structure pages as a result of 'x', things still sucked because 'y'. Libraries z introduced to fix problems, still not great, now here we are with great new thing 'a'.

You probably don't need to get into the immediate grid system and differences with flexbox until you introduce flexbox further down as part of the timeline of improvements.

If however, flexbox and grid are both legitimate options, I'd typically bury that option further down with a 'but what about flexbox' section. Anyone who is engaged enough with the article to think 'what about flexbox' through the whole article will read to the end anyway, and can wait.

tbh, you could pretty much remove the flexbox mention at the start and this introduction would probably be OK.